### PR TITLE
Make SIS use GeoIP 2

### PR DESCRIPTION
**Pull request by markspolakovs**
Untested. Lol.

### CODE REVIEW

File: src/Classes/SIS/SIS_Utils.php [hunk 1]
```diff
@@ -61,13 +63,16 @@ public static function ipLookup($ip)
         $location = [];
 
         if (($query === null) or (pg_num_rows($query) == 0)) {
-            $geoip = geoip_record_by_name($ip);
-            if ($geoip === false) {
+            MyRadioGeoIP::wakeup();
+            try {
+                $geoip = MyRadioGeoIP::getInstance()->city($ip);
+                if (empty($geoip->city)) {
```
Comment (markspolakovs) [inline src/Classes/SIS/SIS_Utils.php:69]: This might need to be an `isset`? Not sure

**Comment by markspolakovs** [comment]
Well... you can open SIS without spewing errors left and right. Don't know if IP resolution actually works though.

**Review by mstratford — COMMENTED** [review]
Looks ok for someone who hasn't actually tested it :)

**Comment by markspolakovs** [comment]
I'll merge it now just to avoid SIS being completely broken, then fix it if need be when the time comes.

**Pull request merged**